src/app/service/poap/DistributePOAP.ts

```diff
@@ -33,10 +35,15 @@ export default async (ctx: CommandContext, guildMember: GuildMember, event?: str
 		],
 		files: [{ name: 'failed_to_send_poaps.csv', attachment: failedPOAPsBuffer }],
 	});
+	if (failedPOAPsList.length <= 0) {
+		Log.debug('all poap successfully delivered');
```

Comment:
'All POAPs successfully delivered.'

Reply:
log messages are generally lower case and without punctuation to help with keeping the cost of storage to a minimum

Reply:
Gotcha 👍